front/components/assistant/AssistantDetails.tsx

```diff
@@ -52,26 +56,24 @@ export function AssistantDetails({
     [doUpdateScope]
   );
 
-  if (!agentConfiguration) {
```

Comment:
Removed the early return, which was causing the element to be unmounted leading to a broken transition on close